sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/WindowedSubscriberFluxWindowIsolatedTest.java

```diff
@@ -318,7 +328,6 @@ public void shouldPickEnqueuedWindowRequestsOnSubscriptionReady() {
         final Upstream<Integer> upstream = new Upstream<>();
 
         final WindowedSubscriber<Integer> subscriber = createSubscriber();
-        upstream.subscribe(subscriber);
```

Comment:
The subscribe() for this test was supposed to be happen only after enqueuing the requests, removing the duplicate subscribe call